projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts

```diff
@@ -60,44 +60,48 @@ export class CommentComponent implements OnChanges, OnDestroy {
   @ViewChild('form') form: ElementRef;
   @ViewChild('editableComment') editableComment: ElementRef<HTMLElement>;
 
-  private subscriptions: Subscription[];
+  private subscriptions: Subscription;
```

Comment:
should it be subscription if it's just one?

Reply:
it can be more in the future I just container